string master_com
---
bool received
